mt-kahypar/partition/deep_multilevel.cpp

```diff
@@ -529,6 +529,7 @@ namespace mt_kahypar {
               context.coarsening.contraction_limit_multiplier);
       context.setupMaximumAllowedNodeWeight(_hg.totalWeight());
       context.setupSparsificationParameters();
+      context.setupThreadsPerFlowSearch();
```

Comment:
can we do this call somewhere else? so it doesn't have to be repeated for each partitioning scheme

Reply:
nope k changes in the deep multilevel scheme